LayoutTests/fast/scrolling/mac/scrollbars/scrollbars-controller-type.html

```diff
@@ -60,6 +60,13 @@
             </div>
         </foreignObject>
     </svg>
+
+    <div style="height: 200px; width: 100px; overflow: scroll; -webkit-overflow-scrolling: touch;" id="disabledScroller">
```

Comment:
`-webkit-overflow-scrolling: touch;` is a no-op on macOS (other than possibly creating a CSS stacking context).

Reply:
Fixed.